logbook/logbook.go

```diff
@@ -789,8 +789,6 @@ func infoFromOp(ref dsref.Ref, op oplog.Op) dsref.VersionInfo {
 		ProfileID:   ref.ProfileID,
 		Name:        ref.Name,
 		Path:        op.Ref,
```

Comment:
Also doesn't feel right, but didn't want to start re-doing this piece without a better understanding of it's impact.

Reply:
Correct, no need to change logbook.